### PR TITLE
Electrical topology session 1

### PR DESCRIPTION
**Pull request by sebastian9**

### CODE REVIEW

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -12661,6 +12664,33 @@
...
+        }
+      },
+      "CurrentFaultSupplyMax": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementInteger"
+          },
+          {
+            "type": "object",
+            "description": "Used for the fault current rating of electrical equipment.",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:12687]:
```suggestion
            "description": "Fault current rating of electrical equipment.",
```

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -12661,6 +12664,33 @@
...
+        "properties": {
+          "CurrentACMax": {
+            "$ref": "#/components/schemas/CurrentACMax"
+          },
+          "CurrentFaultSupplyMax": {
+            "$ref": "#/components/schemas/CurrentFaultSupplyMax"
+          }
+        }
+      },
+      "CurrentFaultSupplyMax": {
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:12680]: Curious why call it CurrentFaultSupplyMax and not CurrentFaultRating?

Comment (sebastian9) [reply Master-OB-OpenAPI.json:12680]: Because equipment such as Energy Storage Systems can have both supply and output fault current.

Comment (cwhanse) [reply Master-OB-OpenAPI.json:12680]: So this is for either Fault or Supply current, right? I was confused by thinking it was "fault supply" as if it was the maximum current that could be supplied to a fault.

**Pull request closed without merging**